### ISSUE TITLE
Automatically splitting drones while importing fits

### ISSUE THREAD

**Issue by molbal** [pyfa-org/Pyfa#2186]
## Quality of life feature request

Hello,

When importing EFT fits with Ctrl+V how hard would it be to automatically split the stack of drones if the ships' max drone bandwidth is full?

Thank you
Veetor

**Comment by DarkFenX** [comment]
It's not hard, but some people (including myself) wouldn't be happy about that. Where do you get the fits where they are not split?

**Comment by molbal** [comment]
I made a workaround in another application so its fine :) 
An example is Gila that has a 100m3 drone bay and 20mbps bandwidth. My fits usually have 6 tech 2 drones and 4 navy drones, like this:


```
[Gila, Gila]

Drone Damage Amplifier II
Drone Damage Amplifier II
Drone Damage Amplifier II

Adaptive Invulnerability Shield Hardener II
Adaptive Invulnerability Shield Hardener II
Federation Navy 10MN Afterburner
Caldari Navy Large Shield Extender
Caldari Navy Large Shield Extender
Caldari Navy Large Shield Extender

Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile
Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile
Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile
Rapid Light Missile Launcher II, Caldari Navy Nova Light Missile
Medium Ghoul Compact Energy Nosferatu

Medium Core Defense Field Purger II
Medium Core Defense Field Purger II
Medium Core Defense Field Purger II


Republic Fleet Valkyrie x4
Valkyrie II x6


Eifyr and Co. 'Rogue' Acceleration Control AC-603
Zainou 'Gnome' Shield Management SM-702
Zainou 'Gnome' Shield Operation SP-903


Nanite Repair Paste x154
```

If I set one of the stacks active it will always use more than 100% bandwidth, requiring me to manually split the stack.

Currently after import it looks like this:
<img width="1052" alt="current after import" src="https://user-images.githubusercontent.com/5754480/81689506-a423d280-945a-11ea-83ba-0e7deb18b7f2.PNG">

This is what my idea is after import:
<img width="1047" alt="proposed after import" src="https://user-images.githubusercontent.com/5754480/81689511-a554ff80-945a-11ea-86e3-bd940007a0fe.PNG">

**Comment by DarkFenX** [comment]
When you make this fit in pyfa, it will limit stacks at 2 drones when you add drones (or add stacks of 2 meds for gila when you hold alt, or stacks of 4 lights), so there is a logic in place for that.

But when fit specifies it as x6, i see little reason to split it into 3 packs of x2. Is it EVE client which exports fit like this?

**Comment by molbal** [comment]
You are right, EVE exports them completely stacked, too. What I meant was to apply that logic immediately after fitting.

At least when I import a fit that's what I do most of the time.

**Comment by molbal** [comment]
If it would not be too much trouble (Low prio) could you enable this functionality with a command line switch please?

**Comment by DarkFenX** [comment]
Proper way would be entry in options, but it's all effort and time. I will try to implement it tomorrow. No promises still :P
